src/frontend/src/common/components/library/data_subview/components/DeleteSamplesConfirmationModal/index.tsx

```diff
@@ -34,10 +33,22 @@ const DeleteSamplesConfirmationModal = ({
 
   const deleteSampleMutation = useDeleteSamples({
     componentOnError: () => {
-      setShouldShowErrorNotification(true);
+      dispatch(addNotification({
+        autoDismiss: true,
+        reduxId: Date.now(),
+        intent: "error",
+        shouldShowCloseButton: true,
+        text: "We were unable to delete the selected samples. Please try again later.",
```

Comment:
Is there a clear line in the sand between when it should get a NotificationComponents `componentKey` vs just be a `text` attribute? I skimmed ahead in the PR and didn't see a clear reason why it goes one way or the other yet, might be nice to document if there's a clear litmus test for being one or the other.

Reply:
Basically, `text` is if your notification only requires text to be shown. if it needs anything more complicated than that, for example if you need to also display a link or a list of IDs, then you make a component for the content that will be displayed in the notification and use `component Key`